mathics/builtin/arithmetic.py

```diff
@@ -1237,6 +1281,42 @@ def test(self, expr) -> bool:
         return isinstance(expr, (Integer, Rational, Real))
 
 
+class RealSign(Builtin):
+    """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/RealAbs.html</url>
+
+    <dl>
+      <dt>'RealSign[$x$]'
+      <dd>returns $-1$, $0$ or $1$ depending on whether $x$ is negative,
+    zero or positive.
+    </dl>
+    'RealSign' is also known as $sgn$ or $signum$ function.
```

Comment:
If it is signum, we should add that before the WMA link a wikipedia link: https://en.wikipedia.org/wiki/Sign_function